.ci_support/update_environment.py

```diff
@@ -8,7 +8,7 @@
 
 import yaml
 
-environment_file = '.ci_support/environment.yml'
+environment_file = 'binder/environment.yml'

```

Comment:
@niklassiemer We have to be careful as for `pyiron_experimental` the mybinder environment is used.

Reply:
Thanks for mentioning! I think we should really unify the repo structure and move/copy to the usual `.ci_support`